src/Profiles/LEV/DataPages/RX/ANTPLUS_LevSpeedSysinfo2.h

```diff
@@ -0,0 +1,17 @@
+#ifndef ANTPLUS_LEVSPEEDSYSINFO2_h
+#define ANTPLUS_LEVSPEEDSYSINFO2_h
+
+#include <Profiles/Lev/DataPages/Base/ANTPLUS_LevBaseMainDataPage.h>
+
+class LevSpeedSysinfo2 : public LevBaseMainDataPage {
```

Comment:
I wonder if its worth making a special intermediate class for SpeedSysInfo and SpeedDistanceInfo so all the fields are not duplicated.

Reply:
Left it unchanged. Please change it later, if you like.